README.md

```diff
@@ -58,6 +59,36 @@ class AdminController < ApplicationController
 end
 ```
 
+## Restricting access to mounted Middleware
+
+Add the gems to your Gemfile:
+
+```ruby
+# Use Balrog for authentication
+gem 'balrog'
+# Use Sidekiq for background jobs
+gem 'sidekiq'
+```
+
+Then disable Sidekiq Web's session in config/initializers/sidekiq.rb
+
+```ruby
+require 'sidekiq/web'
+
+# In order to force sidekiq to use the rails app's session,
+# we need to disable the Sidekiq's session.
+Sidekiq::Web.disable(:sessions)
+```
+
+And lastly, we tell SideKiq::Web to use Balrog::RoutesMiddleware in the config/routes.rb:
```

Comment:
Lead with this bit; explain that you use `.use` (maybe link to https://www.rubydoc.info/gems/rack/Rack%2FBuilder:use) to add Balrog to the 'stack'. For example with `Sidekiq::Web`...